ui/app/templates/components/replication-page.hbs

```diff
@@ -0,0 +1,4 @@
+{{yield (hash 
+  header=(component 'replication-page/replication-header')
+  toggle=(component 'replication-page/replication-toggle')
```

Comment:
does `content=(component 'replicaiton-page/replication-content')` need to be added here?

Reply:
No, I just set it up based on our discussion about have flexibility and a page that just yields.  It's a placeholder, but not needed.  Should I go ahead and remove the whole component (it doesn't do anything) or keep it and just remove it from the replication-page.hbs file?